sql/core/src/test/resources/tpcds-plan-stability/approved-plans-modified/q27.sf100/explain.txt

```diff
@@ -209,208 +209,208 @@ Aggregate Attributes [4]: [avg(agg1#19)#39, avg(UnscaledValue(agg2#20))#40, avg(
 Results [7]: [i_item_id#18, s_state#16, 0 AS g_state#43, avg(agg1#19)#39 AS agg1#44, cast((avg(UnscaledValue(agg2#20))#40 / 100.0) as decimal(11,6)) AS agg2#45, cast((avg(UnscaledValue(agg3#21))#41 / 100.0) as decimal(11,6)) AS agg3#46, cast((avg(UnscaledValue(agg4#22))#42 / 100.0) as decimal(11,6)) AS agg4#47]
 
 (29) Scan parquet spark_catalog.default.store_sales
```

Comment:
It conflicts with `(1) Scan parquet spark_catalog.default.store_sales` and the plan change is valid.

Reply:
`q27` has CTE